src/vcpkg/binarycaching.cpp

```diff
@@ -2095,13 +2109,18 @@ void vcpkg::help_topic_asset_caching(const VcpkgPaths&)
     tbl.format("clear", "Removes all previous sources");
     tbl.format(
         "x-azurl,<url>[,<sas>[,<rw>]]",
-        "Adds an Azure Blob Storage source, optionally using Shared Access Signature validation. URL should "
-        "include "
+        "Adds an Azure Blob Storage source, optionally using Shared Access Signature validation. URL should include "
         "the container path and be terminated with a trailing `/`. SAS, if defined, should be prefixed with a `?`. "
-        "Non-Azure servers will also work if they respond to GET and PUT requests of the form: "
-        "`<url><sha512><sas>`.");
+        "Non-Azure servers will also work if they respond to GET and PUT requests of the form: `<url><sha512><sas>`.");
+    tbl.format("x-script,<template>",
+               "Dispatches to an external tool to fetch the asset. Within the template, \"{url}\" will be replaced by "
+               "the original url, \"{sha512}\" will be replaced by the SHA512 value, and \"{dst}\" will be replaced by "
+               "the output path to save to. These substitutions will all be properly shell escaped, so an example "
+               "template would be: \"curl -L {url} --output {dst}\". \"{{\" will be replaced by \"}\" and \"}}\" will "
+               "be replaced by \"}\" to avoid expansion. Note that this will be executed inside the build environment, "
+               "so the PATH and other environment variables will be modified by the triplet.");
```

Comment:
I don't think fully describing the format is necessarily appropriate for help text. Can this be turned into a link to the docs instead?

I mention this because we're starting to have a problem with our help text being so long that it doesn't fit on a screen

Reply:
(I think it may be good to do that for other entries too. This one is longer than those but they aren't great either)

Reply:
I don't see a problem with the help text output from specifically `vcpkg help <topic>` being too verbose/lengthy; I see it like `git foo -h` versus `git foo --help`: `-h` displays short help (ideally one page, enough to remember key options), and `--help` displays much more verbose reference docs that extensively cover every option in a terse way.